src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+// store.js
```

Comment:
- [ ] Kindly rename this file to `cofigureStore.js` so that your file format meets the project requirements.